### PR TITLE
Fix some grammar mistakes

### PR DESCRIPTION
**Pull request by AtomicCoconut**
I've fixed some grammar mistakes in the README file.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,13 +8,13 @@
...
 ## 🚢 Connect with us
 - Join our 💻 [Discord](https://https://discord.gg/PDWbT7DkPU) for more information about updates and events.
 - You can find our Roadmap [here]()
 
 ## 👷 Development Guide
-AquaHub is a community driven project and we appreciate every help we can get:
+AquaHub is a community driven project, and we appreciate every help we can get:
 - You can suggest new features or report bugs [here](https://github.com/AquaHubStudio/AquaHub/issues).
-- Please read the [Contributing guide](Contributing.md) before constributing to the code.
+- Please read the [contributing guide](Contributing.md) before contributing to the code.
```
Comment (JannisMilz) [inline README.md:20]: I would suggest calling it `contribution guide`

File: README.md [hunk 2]
```diff
@@ -8,13 +8,13 @@
...
+Our mission is to empower everyone to make their vision become reality. AquaHub provides a platform for new **and** experienced project founders to share their projects, connect with other like-minded people and celebrate your success with the community. Everyone on the platform is able to share their opinions and ideas with you to help you build the best project possible. We offer a **friendly and hate-free atmosphere** where everyone is welcome and everyone's opinion matters.
 
 ## 🚢 Connect with us
 - Join our 💻 [Discord](https://https://discord.gg/PDWbT7DkPU) for more information about updates and events.
 - You can find our Roadmap [here]()
 
 ## 👷 Development Guide
-AquaHub is a community driven project and we appreciate every help we can get:
+AquaHub is a community driven project, and we appreciate every help we can get:
```
Comment (JannisMilz) [inline README.md:18]: Is there really a comma? I'm not sure

File: README.md (hunk 1 above)
Comment (JannisMilz) [inline README.md:20]: I would suggest calling it `contribution guide`

File: README.md (hunk 2 above)
Comment (AtomicCoconut) [reply README.md:18]: I think so

File: README.md (hunk 1 above)
Comment (AtomicCoconut) [reply README.md]: done

**Review by JannisMilz — APPROVED** [review]
Thanks for the grammar fixed and additional adjustments. Approved.

**Pull request merged**